clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -240,9 +230,17 @@ export class CopybookDownloadService implements vscode.Disposable {
     private async downloadCopybookFromMFUsingZowe(dataset: string, copybook: string, profileName: string) {
         const copybookPath = createCopybookPath(profileName, dataset, copybook);
         if (!fs.existsSync(copybookPath)) {
-            const content = await this.zoweApi.fetchMember(dataset, copybook, profileName);
-            fs.mkdirSync(createDatasetPath(profileName, dataset), {recursive: true});
-            fs.writeFileSync(copybookPath, content);
+            try {
+                const zoweExplorerApi = ZoweVsCodeExtension.getZoweExplorerApi();
+                const loadedProfile = zoweExplorerApi.getExplorerExtenderApi().getProfilesCache().loadNamedProfile(profileName);
```

Comment:
Can you take an active profile here? The idea is that the end-user can change the active profile in the Explorer and we will work with it.

Reply:
discussed over chat